.github/workflows/beta-release.yml

```
@@ -0,0 +1,356 @@
+name: Beta Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to use'
+        required: true
+        default: 'develop'
+
+      nodeVersion:
+        description: 'Node version'
+        required: true
+        default: '20.9.0'
+        type: string
+
+      betaVersionSuffix:
+        description: 'Beta version suffix (e.g., beta.0, beta.1)'
+        required: true
+        default: 'beta.0'
+        type: string
+
+      includeAngular17:
+        description: 'Include Angular 17 package?'
+        required: false
+        default: 'true'
```

```suggestion
        default: 'false'
```
? Feels like it should be false as default, best practice?